go/exocom-mock/bin/lint

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-set -e
+set -ex
```

Comment:
I think this particular change is unnecessary, and should only be added if there is a good reason. I run this command a lot on my dev machine, and don't want to be bothered with all this logging output.